packages/design-system/src/styles/theme/oc-text.scss

```diff
@@ -13,7 +13,7 @@ html * {
   font-family: var(--oc-font-family);
 }
 
-html, button {
+html {
```

Comment:
Button size is always being controlled inside the button component, no need to handle it here. It seems to interfere when using the design-system as library.